test/validator/remove_validator.js

```diff
@@ -27,6 +27,7 @@ contract('ValidatorSet::removeValidator', (accounts) => {
   let account;
   beforeEach(async () => {
     validatorSet = await ValidatorSet.new();
+    await validatorSet.setupValidator();
```

Comment:
why is the function not `setupValidatorSet()`

Reply:
this will conflict with the correction in #242

Reply:
I have changed `setupValidator()` to `setupValidatorSetDouble()` after going through the PR #242